common/pulp/common/error_codes.py

```diff
@@ -156,3 +156,5 @@
 PLP1014 = Error("PLP1014", _("The configuration value for the key '%(key)s' in "
                              "section '%(section)s' is not valid for the following "
                              "reason: %(reason)s"), ["key", "section", "reason"])
+PLP1015 = Error("PLP1015", _("The required type of JSON data is '%(data_type)s'."),
```

Comment:
Consider rewording `The required type of JSON data is` => `The JSON data must be of type `